content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+*Eviction* is the process by which kubelet proactively fails pods to reclaim
+resources for nodes.
```

Comment:
You could use a glossary definition shortcode here, if you make sure there is a suitable definition in the glossary.
https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary

See https://github.com/kubernetes/website/blame/a22aa8fd4feec091e736bca12b2f8776ade0ac94/content/en/docs/concepts/overview/components.md#L17 for an example of where this is done elsewhere.